### PR TITLE
update docker image

### PR DESCRIPTION
**Pull request by sridhar0605**
Updates docker image for `mapq0` step. 
If you are using this PP `df3ced54b8ea44448d656ecaf42a6829   Somatic WGS June 2019` consider updating your workflow.

### CODE REVIEW

**Comment by chrisamiller** [comment]
Looks to me like the arguments have changed and the CWL might have to change to accommodate that:

Old: https://github.com/genome/docker-mapq0-filter/blob/301de5e65c7f2d3671e35e1e6297110eacb4cdf7/mapq0_vcf_filter.sh

New: https://github.com/genome/docker-mapq0-filter/blob/master/mapq0_vcf_filter.sh

**Review by chrisamiller — APPROVED** [review]
+1 looks good to me

File: definitions/subworkflows/filter_vcf.cwl [hunk 1]
```diff
@@ -69,7 +64,6 @@ steps:
             vcf: filter_vcf_gnomADe_allele_freq/filtered_vcf
             tumor_bam: tumor_bam
             threshold: filter_mapq0_threshold
-            reference: reference
```
[the remark this answers is not in the document]
Comment (sridhar0605) [reply definitions/subworkflows/filter_vcf.cwl:72]: aah yes, in the older PP this was hardcoded  as `TUMOR`.

**Review by tmooney — APPROVED** [review]
+1 🏞️

**Pull request merged**